src/bank.rs

```diff
@@ -145,6 +143,11 @@ impl Bank {
         bank
     }
 
+    pub fn update_subscriptions(&self, subscriptions: &Arc<RpcSubscriptions>) {
```

Comment:
nit: I'd call this function `set_subscriptions`

Reply:
👍